dist/vignette.js

```diff
@@ -318,4 +318,4 @@ var Vignette = (function () {
         return false;
     })();
     return Vignette;
-})();
+}());
```

Comment:
I wonder why these changed... Probably the Typescript compiler was upgraded because we don't have the version locked (`"gulp-typescript": "2.x.x"`)? I don't think we need to worry about it, just noticing :slightly_smiling_face: